app/script/view_model/content/InputBarViewModel.js

```diff
@@ -227,6 +194,22 @@ z.viewModel.content.InputBarViewModel = class InputBarViewModel {
       return conversationEntity.localMessageTimer() && !conversationEntity.hasGlobalMessageTimer();
     });
 
+    this.editedMessage = ko
```

Comment:
This is the new stuff around. The idea is we have a single source of truth to write on the localStorage. Whenever this changes (and we are not editing) we write the content to the localStorage